bubble.js

```diff
@@ -144,7 +144,9 @@ var bubble = {
 		// For an event binding on an object, returns the events that should be bubbled.
 		// For example, `"change" -> ["change"]`.
 		events: function(map, boundEventName) {
-			return map.constructor._bubbleRule(boundEventName, map);
+			if (map && typeof map === 'object') {
```

Comment:
When should bubbling happen? I think there is probably a better way to do this check. Either using `canReflect.isFunctionLike` or `canReflect.isMapLike`.

Reply:
Maybe this should check `!canReflect.isFunctionLike(map)` and the one in `childrenOf` should use `canReflect.isMapLike(child)`?

Reply:
@phillipskevin I made the requested updates.

`canReflect.isMapLike(child)` returns `true` with a function, so I just used `!canReflect.isFunctionLike` for both .

Reply:
So does the `.childrenOf` need ` && child.bind`? What is this check for?

Reply:
I think it is the old way to check if the object is a `can-map`!

Reply:
Ok, this is what I meant to use `isMapLike` for

Reply:
Updated!